### PR TITLE
User profile

### PR DESCRIPTION
**Pull request by WHomer**
What does this PR do?
Satisfies #54

### CODE REVIEW

**Comment by BrennanAyers** [comment]
I think this PR resolves #56 instead of 54?

**Comment by WHomer** [comment]
I can close and reopen if needed, however good catch.

**Review by VinceCarollo — APPROVED** [review]

File: app/controllers/merchants_controller.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class MerchantsController < ApplicationController
+  def index
+  end
 
   def show
```
Comment (BrennanAyers) [inline app/controllers/merchants_controller.rb:5]: 👍

**Review by zachlp2016 — APPROVED** [review]
Ok looks gouda.

**Pull request merged**